### PR TITLE
Update to TspWorldPanel.java to be Mac friendly

### PR DESCRIPTION
**Pull request by JimTyrrell**
Simple change to enable control clicks to work on Macs.  OptaPlanner enablement is fun.

### CODE REVIEW

**Comment by kie-ci10** [comment]
Can one of the admins verify this PR? Comment with 'ok to test' to start the build.

**Comment by yurloc** [comment]
ok to test

File: optaplanner-examples/src/main/java/org/optaplanner/examples/tsp/swingui/TspWorldPanel.java [hunk 1]
```diff
@@ -80,13 +80,13 @@ public void mousePressed(MouseEvent e) {
...
                     double latitude = translator.translateYToLatitude(e.getY());
-                    if (e.getButton() == MouseEvent.BUTTON1) {
+                    if (e.getButton() == MouseEvent.BUTTON1 && !e.isControlDown()) {
                         dragSourceStandstill = TspWorldPanel.this.tspPanel.findNearestStandstill(
                                 new AirLocation(-1L, latitude, longitude));
                         TspSolution tspSolution = TspWorldPanel.this.tspPanel.getSolution();
                         dragTargetStandstill = tspSolution.getDomicile();
                         resetPanel(tspSolution);
-                    } else if (e.getButton() == MouseEvent.BUTTON2 || e.getButton() == MouseEvent.BUTTON3) {
+                    } else if (e.getButton() == MouseEvent.BUTTON2 || e.getButton() == MouseEvent.BUTTON3 ||  && e.isControlDown()) {
```
Comment (ge0ffrey) [inline optaplanner-examples/src/main/java/org/optaplanner/examples/tsp/swingui/TspWorldPanel.java]: Does this compile? It says "|| &&"

**Comment by triceo** [comment]
@ge0ffrey Please close this PR as #538 fully replaces it, with the added benefit of no compilation errors.

**Comment by ge0ffrey** [comment]
Closing this in favor of Lukas's fix of this PR which is merged: https://github.com/kiegroup/optaplanner/pull/538
@JimTyrrell Would you mind testing out master with Lukas's changes on a Mac? None of us have a Mac.

**Pull request closed without merging**